modules/oc-mirror-workflows-fully-disconnected-v2.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-workflows-fully-disconnected-v2_{context}"]
+= Mirroring an image set in a fully disconnected environment
+
+You can mirror image sets in a fully disconnected environment where the {product-title} cluster cannot access the public internet.
+
+.Procedure
+
+. *Mirror to disk*: Prepare an archive containing the image set for mirroring. Internet access is required.
+
+. *Manual step*: Transfer the archive to the network of the disconnected mirror registry.
+
+. *Disk to mirror*: To mirror the image set from the archive to the target disconnected registry, run oc-mirror plugin v2 from the environment that has access to the mirror registry.
```

Comment:
I am a bit sceptical about this one?. 
*Mirror to disk*: Prepare an archive containing the image set for mirroring. Internet access is required.
. *Manual step*: Transfer the archive to the network of the disconnected mirror registry.
. *Disk to mirror*: To mirror the image set from the archive to the target disconnected registry, run oc-mirror plugin v2 from the environment that has access to the mirror registry.

s/ 
. Prepare an archive containing the image set for mirroring. Internet access is required.

. Transfer the archive to the network of the disconnected mirror registry.

. To mirror the image set from the archive to the target disconnected registry, run oc-mirror plugin v2 from the environment that has access to the mirror registry.

Reply:
The update is fine, but we do need to mention about mirror to disk and disk to mirror here.